factory/processing/processComponents.go

```diff
@@ -624,9 +626,19 @@ func (pcf *processComponentsFactory) Create() (*processComponents, error) {
 		return nil, fmt.Errorf("%w when assembling components for the sent signatures tracker", err)
 	}
 
-	executionResultsTracker := executionTrack.NewExecutionResultsTracker()
 	blocksQueue := queue.NewBlocksQueue()
-	blocksQueue.RegisterEvictionSubscriber(executionResultsTracker)
+	executionResultsTracker := executionTrack.NewExecutionResultsTracker()
```

Comment:
[question]
Is this intentionally removed?
`blocksQueue.RegisterEvictionSubscriber(executionResultsTracker)`